src/HttpCall/Request/Goutte.php

```diff
@@ -20,25 +20,13 @@ public function send($method, $url, $parameters = [], $files = [], $content = nu
         return $page;
```

Comment:
```
     public function send($method, $url, $parameters = [], $files = [], $content = null, $headers = [])
```

`$headers` are never used here. I propose either to check if `$headers` is not empty and merge array with `$this->requestHeaders` or use `$headers` instead in this case.

It can be useful for tests, where you can set multiple headers in table.